plugins/foreman_ansible/1.x/index.md

```diff
@@ -46,12 +46,14 @@ For Ansible versions older than 2.2:
 
 You can configure it via the following environment variables:
 
-* FOREMAN_URL: the URL of your Foreman installation (default "http://localhost:3000")
+* FOREMAN_URL: the URL of your Foreman installation (default "http://localhost:3000") - change it to your Foreman URL, e.g: "https://myforeman.mylocal.lan", it should not be port 3000.
 * FOREMAN_SSL_CERT: The public key when using SSL client certificates (default "/etc/foreman/client_cert.pem")
 * FOREMAN_SSL_KEY: The private key when using SSL client certificates (default  "/etc/foreman/client_key.pem")
 * FOREMAN_SSL_VERIFY: wether to verify SSL certificates. Use *False*
   to disable certificate checks. You can also set it to CA bundle (default is "True").
 
+If you are using the Katello plugin, the SSL certificate and key are set at `/etc/httpd/conf.d/05-katello-ssl.conf`. SSLCertificateFile, SSLCertificateKeyFile contain the values for FOREMAN_SSL_CERT and FOREMAN_SSL_KEY. By default these are at `/etc/pki/katello/certs/katello-default-ca.crt` and `/etc/pki/katello/private/katello-default-ca.key`.
```

Comment:
Are you sure this is true? On my katello nightly box `/etc/httpd/conf.d/05-katello-ssl.conf` doesn't exist. AFAIK katello uses the same apache file, but different values.